### PR TITLE
OTWO-5507 Fixed Errbit error- no implicit conversion of nil into String

### PR DESCRIPTION
**Pull request by Priya5**

### CODE REVIEW

File: app/views/enlistments/index.xml.builder [hunk 1]
```diff
@@ -4,7 +4,9 @@ xml.response do
   xml.items_returned @enlistments.length
   xml.items_available @enlistments.total_entries
   xml.first_item_position @enlistments.offset
-  xml.result do
-    xml << render(partial: 'enlistment', collection: @enlistments, locals: { builder: xml })
+  if @enlistments.present?
+    xml.result do
+      xml << render(partial: 'enlistment', collection: @enlistments, locals: { builder: xml })
+    end
   end
```
Comment (PDegenPortnoy) [inline app/views/enlistments/index.xml.builder]: Should there be a valid XML response when `@enlistments.present?` is a falsy value?

Comment (Priya5) [reply app/views/enlistments/index.xml.builder]: We should return a valid responsę, as it is a valid request without any error. 
In this case, when `@enlistments.present?` is a falsy value, the response will not contain `result` element.
```
<?xml version="1.0" encoding="UTF-8"?>
<response>
    <status>success</status>
    <items_returned>0</items_returned>
    <items_available>0</items_available>
    <first_item_position>0</first_item_position>
</response>
```

Comment (PDegenPortnoy) [reply app/views/enlistments/index.xml.builder]: Perfect; thank you!

**Comment by PDegenPortnoy** [comment]
ok to test

**Comment by PDegenPortnoy** [comment]
Retest this please

**Comment by Priya5** [comment]
retest this please

**Comment by Priya5** [comment]
retest this please

**Pull request merged**